### ISSUE TITLE
potential memory leak

### ISSUE THREAD

**Issue by jbphet** [phetsims/vibe#22]
While investigating #20, I found that after extensive fuzz testing the demo/test app (i.e. vibe_en.html) would crash in Chrome with the "Aw, Snap!" error.  This could indicate a memory leak.  I ran fuzz testing with ```fuzzMouse=100``` and took heap snapshots, sorted the data by the "object" column and the "Retained Size" column, and it does appear that something is leaking.  Screenshots below.

At time ~0:
![vibe-mem-leak-1](https://cloud.githubusercontent.com/assets/3082831/11192582/23d92f60-8c5f-11e5-9ec7-49a91f8f7762.png)
![vibe-mem-leak-2](https://cloud.githubusercontent.com/assets/3082831/11192587/2970a340-8c5f-11e5-9d11-cf83eabdba7b.png)

At time ~3 min:
![vibe-mem-leak-3](https://cloud.githubusercontent.com/assets/3082831/11192596/343ceae0-8c5f-11e5-8c9d-590ee016e0e4.png)
![vibe-mem-leak-4](https://cloud.githubusercontent.com/assets/3082831/11192600/3b231578-8c5f-11e5-9ee0-628463dec7e0.png)

At time ~6 min:
![vibe-mem-leak-5](https://cloud.githubusercontent.com/assets/3082831/11192607/46c97660-8c5f-11e5-8acc-635a57ce8e00.png)
![vibe-mem-leak-6](https://cloud.githubusercontent.com/assets/3082831/11192613/4c5b78a8-8c5f-11e5-9437-eee1beeef90e.png)

**Comment by jbphet** [comment]
Memory leaks have been found that may explain what I was seeing here, and that don't relate to vibe, see https://github.com/phetsims/tasks/issues/546, https://github.com/phetsims/joist/issues/327, and https://github.com/phetsims/joist/issues/329.

**Comment by jbphet** [comment]
I tested the master version, which includes a fix for worst memory leak linked in the comment above, and there was far less leaked memory.  The leakage that is seen now is similar to and probably explainable by the issue noted in https://github.com/phetsims/joist/issues/329.  Here is a screenshot where snapshots were taken at roughly t=0 min, 3 min, and 6 min, as was done above.
![vibe-memory](https://cloud.githubusercontent.com/assets/3082831/14151630/1018b03c-f66c-11e5-87a0-006305bdb711.PNG)
From this image, it can be seen that the rate of leakage nothing like it was during the original testing.  Closing, since this is most likely not a problem with vibe.
